tools/sylph/sylph_query.xml

```diff
@@ -0,0 +1,232 @@
+        <conditional name="sketch_reads">
+            <param name="type_reads" type="select" label="Select the type of reads used">
+                <option value="single_fastq">Individual single-end reads</option>
+                <option value="single_group_fastq">Group of single-ended reads</option>
+                <option value="paired">One set of paired-end reads</option>
+                <option value="paired_group"> Group of paired-ended reads</option>
+            </param>
+        <!-- Only permitting fastq as tool input only allows fastq and fastq.gz as file ext -->
+            <when value="single_fastq">
+                <param name="input" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Single-end input reads"/>
+            </when>
+            <when value="single_group_fastq">
+                <param name="input" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Single-end input reads" multiple="true"/>
+            </when>            
+            <when value="paired">
+                <param name="input_1" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads 1"/>
+                <param name="input_2" type="data" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads 2"/>
+            </when>
+            <when value="paired_group">
+                <param name="input" type="data_collection" format="fastq,fastq.gz,fastqsanger,fastqsanger.gz" label="Paired-end input reads" collection_type="paired"/>
+            </when>
+        </conditional>
+        <conditional name="database_select">
+            <param name="select" type="select" label="Choose the source for databases and metadata">
+                <option value="cached">Cached data</option>
+                <option value="custom">Custom</option>
+                <option value="cached_and_custom">Cached and custom databases</option>
+            </param>
+            <when value="cached">
+                <param label="Select a sylph database" name="sylph_database" type="select" multiple="true" optional="false">
+                    <options from_data_table="sylph_databases">
+                        <validator message="No Sylph databases are available" type="no_options" />
+                    </options>
+                </param>
+            </when>
+            <expand macro="input_database"/>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data format="tabular" name="output" label="${tool.name} on ${on_string}"/>
+    </outputs>
+    <tests>
+        <!-- Test 1 Single-end Read ANI -->
+        <test expect_num_outputs="1">
+            <conditional name="database_select">
+                <param name="select" value="custom"/>
+                    <conditional name="sketch_fasta">
+                        <param name="type_fasta" value="single_fasta"/>
+                        <param name="input" value="e.coli-o157.fasta.gz" ftype="fasta"/>
```

Comment:
You could use remote files here, this way we don't need to put it into the repo. Have a look at `location` https://docs.galaxyproject.org/en/latest/dev/schema.html#id108